src/v/transform/api.cc

```diff
@@ -446,6 +450,23 @@ ss::future<> service::start() {
     co_await _batcher->start();
     co_await _manager->start();
     register_notifications();
+
+    _log_manager = std::make_unique<logging::manager<ss::lowres_clock>>(
+      _self,
+      std::make_unique<logging::rpc_client>(
+        &_rpc_client->local(),
+        &_topic_table->local(),
+        &_metadata_cache->local()),
+      config::shard_local_cfg().data_transforms_logging_buffer_capacity_bytes(),
+      config::shard_local_cfg().data_transforms_logging_line_max_bytes.bind(),
+      config::shard_local_cfg()
+        .data_transforms_logging_flush_interval_ms.bind());
+    auto fut = co_await ss::coroutine::as_future(_log_manager->start());
+    vassert(
+      !fut.failed(),
+      "Failed to start transform::logging::manager: {}",
+      fut.get_exception());
+    fut.ignore_ready_future();
```

Comment:
Why abort the process here? Just return the failed future and the system will start a graceful shutdown sequence.

Reply:
oh, duh 🤦